res/values/strings.xml

```diff
@@ -123,6 +123,7 @@
     <string name="common_remove_upload">Remove upload</string>
     <string name="common_retry_upload">Retry upload</string>
     <string name="common_cancel_sync">Cancel sync</string>
+    <string name="common_cancel">Cancel</string>
```

Comment:
This is no longer needed, it was deleted in https://github.com/owncloud/android/pull/2275/files#diff-4691bafd7791e471b30c492e6f43d65bL130